logger.go

```diff
@@ -241,8 +242,8 @@ func (l *Logger) NoSendFatal() {
 	}
 }
 
-func (l *Logger) Log(level Level, args ...interface{}) {
-	if l.check(level) {
+func (l *Logger) Log(ctx context.Context, level Level, args ...interface{}) {
```

Comment:
Currently, we are not utilizing the context for anything in particular, but we would like to extract the internal values from the context in the future.